system/Commands/Utilities/Routes/AutoRouteCollector.php

```diff
@@ -36,6 +36,7 @@ public function __construct(string $namespace, string $defaultController, string
 
     /**
      * @return array<int, array<int, string>>
+     *
```

Comment:
How configurable are these? Some they are both "return" annotations I like having them together, but I totally understand if CS Fixer sees them as two separate annotation types and I would rather keep the rule than forsake it just for this.

Reply:
You're probably looking for the `groups` option of the `phpdoc_separation` rule. I'm on mobile so I can't show you yet but you can run `vendor/bin/php-cs-fixer describe phpdoc_separation` to see the details.

Reply:
https://cs.symfony.com/doc/rules/phpdoc/phpdoc_separation.html

Reply:
Yes, precisely! I would be in favor of grouping these (and any vendor-specific prefixes) together with the native annotations. I'm open to other opinions, but that seems the cleanest and easiest to understand at a glance.

Reply:
See latest commit https://github.com/codeigniter4/CodeIgniter4/pull/6471/commits/4369d209b7b59cd882e8b9bcd293f2df66eb4df7

Reply:
Wow, thanks for that fast turnaround Paul! I just sat down to look at this. ❤️